server/bot/commands/trivia.ts

```diff
@@ -0,0 +1,266 @@
+import { Message, GuildTextableChannel, TextableChannel, User, EmbedOptions, Client } from 'eris'
+import { Command, DB } from '../imports/types'
+import { getInsult } from '../imports/tools'
+import fs from 'fs'
+
+async function parseTriviaList (fileName: string) {
+  const data = await fs.promises.readFile(`./triviaLists/${fileName}.txt`, 'utf8')
+  const triviaList = data.split('\n')
+  return triviaList
+    .map(el => el.replace('\n', '').split('`'))
+    .filter(el => el.length >= 2 && el[0])
+    .map(el => ({
+      question: el[0],
+      answers: el.slice(1).map(ans => ans.trim())
+    }))
+}
+
+export class TriviaSession {
+  revealMessages = ['I know this: ', 'Easy: ', 'Of course, it\'s: ']
+  failMessages = ['You suck at this', 'That was trivial, really', 'Moving on...']
+  settings: { maxScore: number, timeout: number, delay: number, botPlays: boolean, revealAnswer: boolean }
```

Comment:
Make changes elsewhere in the code to accommodate for these renames.
```suggestion
  settings: { maxScore: number, timeout: number, timeLimit: number, botPlays: boolean, revealAnswer: boolean }
```